src/app/WeatherApp.tsx

```diff
@@ -0,0 +1,62 @@
+import { Col, Row } from "antd";
+import { FaDroplet as HumidityIcon } from "react-icons/fa6";
+import { FaWind } from "react-icons/fa";
+import WeatherIcons from "./weatherIcons";
+import getDate from "./getDate";
+import { useState } from "react";
+
+const WeatherApp: React.FC<{
+  res: {
+    data: {
+      main: { temp: string; humidity: string };
+      name: string;
+      wind: { speed: string };
+      weather: Array<{ main: string }>;
+    };
+  } | null;
+}> = ({ res }) => {
+  if (res === null) {
+    console.log("null");
+  } else {
+    const tempInCalvin = res.data.main.temp;
+    const tempInCelsius = Math.round(Number(tempInCalvin) - 273.15);
+
+    const [changeTime, setChangeTime] = useState(getDate());
```

Comment:
Misleading variable name, if I understand correctly this is current time, up to 1 second thanks to the setInterval.